src/applications/vaos/components/layout/DetailPageLayout.jsx

```diff
@@ -88,14 +83,7 @@ function CancelButton({ appointment }) {
     />
   );
 
-  if (
-    APPOINTMENT_STATUS.cancelled !== status &&
-    !isCommunityCare &&
-    !isCompAndPenAppointment &&
-    !isVideo &&
-    !isPastAppointment
-  )
-    return button;
```

Comment:
Removing the cancellable logic from the frontend since the logic is now placed in [vets-api layer to set CnP, COVID, CC and telehealth (video) appointment cancellable property to false](https://github.com/department-of-veterans-affairs/va.gov-team#workspaces/appointments-team-603fdef281af6500110a1691/issues/zh/326)